QGCExternalLibs.pri

```diff
@@ -79,20 +79,20 @@ INCLUDEPATH += libs/qwt
 # Uninstalling from Linux can be done with `sudo make uninstall`.
 #
 XBEE_DEPENDENT_HEADERS += \
-	src/comm/XbeeLinkInterface.h \
-	src/comm/XbeeLink.h \
-	src/comm/HexSpinBox.h \
-	src/ui/XbeeConfigurationWindow.h \
-	src/comm/CallConv.h
+    src/comm/XbeeLinkInterface.h \
```

Comment:
> Have you tabs gone wonky? Not sure if that is a tab stop or your tabs are set to 8 instead of 4?

Hmm... It's the other way around. I deleted my Qt Creator settings the other day and it reverted to *Fix Whole Document*. When I edited that file and saved it, it *Fixed* all the original tabs by replacing it with spaces (or the proper number of spaces).

![screen shot 2015-10-22 at 3 40 32 pm](https://cloud.githubusercontent.com/assets/749243/10676360/a8d1bb28-78d3-11e5-878a-3ca64dd92be1.png)